torch/csrc/distributed/rpc/rref_impl.h

```diff
@@ -395,6 +399,28 @@ class TORCH_API OwnerRRef final : public RRef {
   friend class RRefContext;
 
   std::shared_ptr<JitFuture> future_;
+
+ public:
+  // Records an event per each stream in the context and stores them in
+  // the current OwnerRRef instance if CUDA is available,
+  // otherwise does nothing.
+  void recordAllStreams(std::shared_ptr<LazyStreamContext> ctx);
+
+  // Blocks all streams in the context on all events previously stored in
+  // the current OwnerRRef instance if CUDA is available,
+  // otherwise does nothing.
+  void blockAllStreams(std::shared_ptr<LazyStreamContext> ctx);
+ private:
+
+ #ifdef USE_CUDA_NOT_ROCM
```

Comment:
BTW, looks like there is an unintentional space before the #ifdef

Reply:
addressed